x/accounts/defaults/lockup/README.md

```diff
@@ -36,6 +53,133 @@ type BaseLockup struct {
+
+Clawback are an extension to normal lockup account, the differances between a clawback enabled account and normal account is the BaseAccount interface implementer passed during account creation.
+
+Clawback enabled lockup account will be able to trigger a clawback execution by the admin to claw the remain locked token in the lockup account. Other execution still need to be triggered by the account owner. It will not be able to perform actions such as delegate, undelegate, or withdraw rewards like the default lockup account.
+
+```mermaid
+flowchart TD 
+    A[lockup account's BaseAccount] -->|type casting| B{Is BaseClawback?}
+
+    B -->|TRUE| C([Clawback enabled account])
+
+    B -->|FALSE| D([Clawback not enabled account])
+
+    C --> 1[Withdraw Unlocked Tokens]
+    C --> 2[Send Coins]
+    C --> 3[Clawback tokens]
+
+    D --> 1[Withdraw Unlocked Tokens]
+    D --> 2[Send Coins]
+    D --> 4[Delegate]
+    D --> 5[Undelegate]
+    D --> 6[Withdraw Reward]
+```
+
+### How to enable clawback
+
+accounts module keeper takes multiple AccountCreatorFunc as parameter, each represents an account type.
+
+```go
+func NewKeeper(
+	cdc codec.Codec,
+	env appmodule.Environment,
+	addressCodec address.Codec,
+	ir InterfaceRegistry,
+	accounts ...accountstd.AccountCreatorFunc,
+) (Keeper, error) {...}
+```
+
+All lockup account type AccountCreatorFunc accept a boolean typed flag named clawbackEnable, if true the BaseClawback will be passed as BaseAccount and account type name with additional CLAWBACK_ENABLE_SUFFIX else BaseLockup will be passed in instead and account type name will be kept as default.
+
+Example, in ContinuousLockingAccount:
+
+```go
+func NewContinuousLockingAccount(clawbackEnable bool) accountstd.AccountCreatorFunc {
+	return func(d accountstd.Dependencies) (string, accountstd.Interface, error) {
+		if clawbackEnable {
+			baseClawback := newBaseClawback(d)
+
+			return types.CONTINUOUS_LOCKING_ACCOUNT + types.CLAWBACK_ENABLE_SUFFIX, ContinuousLockingAccount{
+				BaseAccount: baseClawback,
+				StartTime:   collections.NewItem(d.SchemaBuilder, types.StartTimePrefix, "start_time", collcodec.KeyToValueCodec[time.Time](sdk.TimeKey)),
+			}, nil
+		}
+
+		baseLockup := newBaseLockup(d)
+		return types.CONTINUOUS_LOCKING_ACCOUNT, ContinuousLockingAccount{
+			BaseAccount: baseLockup,
+			StartTime:   collections.NewItem(d.SchemaBuilder, types.StartTimePrefix, "start_time", collcodec.KeyToValueCodec[time.Time](sdk.TimeKey)),
+		}, nil
+	}
+}
+```
+
+```mermaid
+flowchart TD 
+    subgraph accounts module keeper initialization
+    A[NewContinuousLockingAccount] -->|passed clawbackEnable as parameter| B{Is clawbackEnable?}
+
+    subgraph C [CONTINUOUS_LOCKING_ACCOUNT + CLAWBACK_ENABLE_SUFFIX]
+    E[Passed BaseClawback to ContinuousLockingAccount struct]
+    end
+
+    subgraph D [CONTINUOUS_LOCKING_ACCOUNT]
+    F[Passed BaseLockup to ContinuousLockingAccount struct]
+    end
+
+    B -->|TRUE| C
+    B -->|FALSE| D
+
+    end
+```
```

Comment:
```suggestion
```
this doesnt really show much, feel like its noise to the readme

Reply:
removed